py/redrock/zfind.py

```diff
@@ -546,13 +588,20 @@ def zfind(targets, templates, mp_procs=1, nminima=3, archetypes=None, priors=Non
             del allresults[tid]['meta']
 
         # Standardize column sizes
-        if allzfit['subtype'].dtype != '<U20':
-            allzfit.replace_column('subtype', allzfit['subtype'].astype('<U20'))
+        if allzfit['subtype'].dtype != '<U32':
+            allzfit.replace_column('subtype', allzfit['subtype'].astype('<U32'))
```

Comment:
What's the case where SUBTYPE needs to be 32 characters instead of 20?  I don't see any subtypes in the current files that are that long, and this changes the datamodel of the output files.

Reply:
I changed it to 'U32' if we ever use the nearest neighbours. In that case, the subtypes can be longer.